public/data/Ranamalsingh12.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Rana Mal Singh",
+  "type": "personal",
+  "bio": "Front-end Developer | Currently Learning Node.js | Web3 Enthusiast! | Open Source Enthusiast!",
+  "avatar": "https://github.com/Ranamalsingh12.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Ranamalsingh12",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/RanaMS999",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

Comment:
I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant